velox/functions/prestosql/types/TDigestType.cpp

```diff
@@ -17,6 +17,18 @@
 
 namespace facebook::velox {
 
+folly::dynamic TDigestType::serialize() const {
+  folly::dynamic obj = folly::dynamic::object;
+  obj["name"] = "Type";
+  obj["type"] = name();
+  folly::dynamic children = folly::dynamic::array;
+  for (auto& param : parameters_) {
+    children.push_back(param.type->serialize());
+  }
+  obj["cTypes"] = children;
```

Comment:
```
nit: s/cTypes/childTypes/
```

Reply:
That name is already used by the serialization framework, not sure how dangerous is if we change it now